.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Request a new feature
+title: "[Feature-Request]"
```

Comment:
Add also `{Title of feature request}`.
Same for BUG template.
People tend to leave it as it is, with no distinguishable title.